src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="HOST/host.test.corefx.net"
+principal2="HTTP"
+krb_user="krb_user"
+password="password"
```

Comment:
Thanks for confirming, @rahulkotecha They probably all need passwd entry that maps to the Kerberos principal. One minor change we should make is to use non-standard service principal names. eg. change HTTP to TestHTTP etc. This will further reduce probability of a regular service using the service principals we are creating.

cc: @joshfree 
@morganbr , @joshfree, can you recommend some Unix security expert who can validate our thinking?